arrays.js

```diff
@@ -0,0 +1,50 @@
+const colors=[ "orange", "blue", "green", "black"];
+//  for (const color of colors){
+//      console.log(color);
+//  }
+
+// colors.forEach(item => console.log(item));
+
+// let number = 3;
+// function rotateRight(number, colors){
+//     for(i=0; i<number; i++){
+//         let last = colors.pop();
+//         colors.unshift(last);
+//     }
+// }
```

Comment:
Lepa funkcija. Ono sto smo mogli da odradimo kako bi je jos ulepsali jeste da zamenimo ovo let last, da ne bude inicijalizovano unutar for petlje. 
```suggestion
// function rotateRight(number, colors){
//      let last;
//     for(i=0; i<number; i++){
//         last = colors.pop();
//         colors.unshift(last);
//     }
// }
```

Javascript generalno lepo upravlja memorijom i cisti sama za sobom u vecini slucajeva, doduse ovo bi nam mogao biti improvement koji smanjuje koriscenje memorije i ubrzava izvrsavanje jer se manje procesa izvrsava